swap_meet/clothing.py

```diff
@@ -1,2 +1,10 @@
-class Clothing:
-    pass
\ No newline at end of file
+from .item import Item
+class Clothing(Item):
+    def __init__(self, id = None, fabric = "Unknown", condition = 0, age = None):
```

Comment:
```suggestion
    def __init__(self, id=None, fabric="Unknown", condition=0, age=None):
```

Spaces do not surround `=` when you're using default params.